srcpkgs/pantheon-session/files/pantheon-mimeapps.list

```diff
@@ -0,0 +1,167 @@
+[Default Applications]
```

Comment:
Where is this list coming from? Would it not make more sense to put this in the default settings package?

Reply:
You mean, i should remove `pantheon-mimeapps.list`?